torch/utils/dlpack.py

```diff
@@ -25,3 +30,37 @@
 The dlpack shares the tensors memory.
 Note that each dlpack can only be consumed once.
 """)
+
+# TODO: add a typing.Protocol to be able to tell Mypy that only objects with
+# __dlpack__ and __dlpack_device__ methods are accepted.
+def from_dlpack(ext_tensor: Any) -> torch.Tensor:
+    """from_dlpack(ext_tensor) -> Tensor
+
+    Convers a tensor from a external library into a ``torch.Tensor``
+    by means of the ``__dlpack__`` protocol.
+
+    The tensor will share the memory with the object represented
+    in the dlpack.
+
+    Note that each dlpack capsule can only be consumed once. Otherwise
```

Comment:
Here and elsewhere we should be careful to use the proper name "DLPack" consistently